### ISSUE TITLE
Context menus from '...' buttons misplaced after scrolling

### ISSUE THREAD

**Issue by Toff-kun** [janch32/Spotify-WPW#7]
Scrolling a song list down and then tapping on any of the '...' buttons makes the context menus open where they would open if the content hasn't been scrolled down instead of opening right next to the tapped button. The distance between the button and the menu further increases the more you scroll down until the menu starts opening beyond the screen, making it look like the button doesn't work.
Tapping and holding on a song instead of tapping the '...' button opens the menu where it should be.
I couldn't reproduce this problem on my desktop browsers with a mouse.

![image](https://user-images.githubusercontent.com/10385850/54835510-e2948600-4cca-11e9-88fe-51bba4006299.png)

![image](https://user-images.githubusercontent.com/10385850/54835531-f50ebf80-4cca-11e9-843f-558b0fbb9dfe.png)

**Comment by janchaloupka** [comment]
Does it also happen when you touch and hold your finger on the song name (same thing as mouse right-click)?

**Comment by Toff-kun** [comment]
No, I posted two screens, second one is touch and hold, and this way it opens where it's supposed to, as you can see. I suspected it's related to touch events, probably different way coordinates are calculated for those?

**Comment by Toff-kun** [comment]
Also a minor inconvenience but probably related to this in some way. It's possible to hold and drag the volume slider with a mouse, but with touch it's only possible to move it by tapping. Can't move the slider by sliding my finger.

**Comment by Toff-kun** [comment]
And that's about all the bugs I could find. :)

**Comment by janchaloupka** [comment]
So the easiest fix would be to remove the three dots and rely on the "touch and hold" 😄.
But I suspect it is just some misaligned element.

**Comment by Toff-kun** [comment]
Managed to fix this by changing `position: fixed` to `position: absolute` for `react-contextmenu`, however this breaks context menu positioning for mouse input / long touch instead. It would be better to disable long touch completely, but it's rather complicated to implement. It shouldn't affect normal usage though, as not many users will try to use long presses.
I added it to the script instead of the css to only change position when touch input is detected in case you decide to make it available for desktops as well, as some users requested it.
